packages/experiments/src/components/Chiclet/ChicletXsmall.styles.tsx

```diff
@@ -0,0 +1,78 @@
+import { normalize } from '../../Styling';
+import { IChicletXsmallStyleProps, IChicletXsmallStyles } from './ChicletXsmall.types';
+
+export const getStyles = (props: IChicletXsmallStyleProps): IChicletXsmallStyles => {
+  const { theme, className } = props;
+  const { palette } = theme;
+
+  return {
+    root: [
+      'ms-ChicletXsmall',
```

Comment:
@natalieethell would it be preferable to hoist this into a global classnames object?